lib/src/test/kotlin/WeatherSensorService/ClosedRangeExtension.test.kt

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
To be consistent (and this is pretty common), ClosedRangeExtension.test.kt should be ClosedRangeExtensionTests.kt